src/Stripe.net/Services/SubscriptionSchedules/SubscriptionScheduleListOptions.cs

```diff
@@ -0,0 +1,39 @@
+namespace Stripe
+{
+    using System;
+    using Newtonsoft.Json;
+
+    public class SubscriptionScheduleListOptions : ListOptionsWithCreated
+    {
+        // TODO: Figure out how to document those
```

Comment:
How do I document those? Do I ignore since you're fixing in the next major revision?